packages/webapp/src/components/Navigation/NavBar/styles.module.scss

```diff
@@ -37,6 +39,29 @@
   background: white;
   box-shadow: none;
   height: var(--global-navbar-height);
+
+  .drawer {
+    background-color: var(--teal700);
+  }
+
+  .drawerHeader {
+    svg {
+      path {
+        color: var(--teal50);
+      }
+    }
+  }
+
+  .drawerContent {
+    padding: 0;
+    margin-bottom: -16px;
+  }
+
+  @include desktop-breakpoint {
```

Comment:
No action needed but I was curious -- For specifically the display:none css (not other styling) I wondered what you think about the MUI useTheme() hook that was already in Navbar.  It seemed useful to conditionally render based on breakpoints. I expanded its use from what it was before and it seems like a handy and explicit way to show what is going on in the JS versus CSS to make things mobile/responsive.

Reply:
I really like the idea of going the JSX route to hide/show stuff, I agree it seems a lot easier to read. I'll refactor!